include/triton-shared/Dialect/TritonStructured/IR/TritonStructuredDialect.td

```diff
@@ -116,7 +128,7 @@ def TTS_LoadOp : TTS_Op<"load", [
   let summary = "optionally load data from in memory to fill a portion of the tensor";
 
   let arguments = (ins TT_PtrLike:$ptr,
```

Comment:
does it make sense to name the variable rather than use a comment? i.e. `$maskDims`